sdk/storage/Azure.Storage.Blobs/src/Models/Internal/BlobRequestConditionProperty.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Blobs.Models
```

Comment:
Models.Internal

Reply:
None of our other internal models are actually in the Models.Internal namespace